packages/address/src/AddressFormatter.ts

```diff
@@ -1,15 +1,8 @@
 import {Address, FieldName, Country} from '@shopify/address-consts';
 
-import {renderLineTemplate, FIELDS_MAPPING} from './utilities';
+import {formatAddress, buildOrderedFields} from './utilities';
 import {loadCountry, loadCountries} from './loader';
 
-const FIELD_REGEXP = /({\w+})/g;
-const LINE_DELIMITER = '_';
-const DEFAULT_FORM_LAYOUT =
-  '{firstName}{lastName}_{company}_{address1}_{address2}_{city}_{country}{province}{zip}_{phone}';
-const DEFAULT_SHOW_LAYOUT =
```

Comment:
These were duplicated in utilities. Now that the actual logic lives in utilities, they're no longer needed here.